framework/helpers/BaseStringHelper.php

```diff
@@ -45,6 +45,23 @@ public static function byteSubstr($string, $start, $length = null)
     {
         return mb_substr($string, $start, $length === null ? mb_strlen($string, '8bit') : $length, '8bit');
     }
+    
+    /**
+     * Returns given string with first character uppercased
+     * @param string $string
+     * @param string $encoding
+     * @return string
+     */
+    public static function ucfirst($string, $encoding = null)
+    {
+        if (!$encoding) {
+            $encoding = \mb_internal_encoding();
+        }
+        $strlen = mb_strlen($string, $encoding);
+        $firstChar = mb_substr($string, 0, 1, $encoding);
+        $then = mb_substr($string, 1, $strlen - 1, $encoding);
```

Comment:
👎 for `$then` variable name. Be more specific. Also using noun is more preferable.